### ISSUE TITLE
End Session Endpoint not found  /endsession/s

### ISSUE THREAD

**Issue by dallasbeek** [DuendeSoftware/Support#703]
**Which version of Duende IdentityServer are you using?**
Business Edition 6.2.3

**Which version of .NET are you using?**
net7.0

**Describe the bug**
See a number of reports in my logs where a 404 is surfacing when ending sessions.  The path of the URL is
https://**********/connect/endsession/s?sid=*******************&iss=https%3A%2F%2F*****.*****.com

My client libraries are
vuex-oidc 4.0.2
oidc-client-ts 2.2.2

On my client I have a inactivity timeout module that makes a call to
this.signOutOidc();

I haven't determined if this is the source of problem, seems to work most of the time.
I'm pretty sure the endpoint is not supposed to include /s. Any ideas why this happens or how to fix?

**To Reproduce**
Haven't determined how to reproduce.

Steps to reproduce the behavior.

**Expected behavior**

**Log output/exception with stacktrace**

```
data
```

**Additional context**

**Comment by josephdecock** [comment]
>  I'm pretty sure the endpoint is not supposed to include /s. Any ideas why this happens or how to fix?

That's correct, the endsession endpoint does not include that /s, and our discovery endpoint should never include the /s. If it ever does, that is a bug, but that would be pretty surprising. I suspect that this is a bug in the client side code with constructor the end session url.

**Comment by josephdecock** [comment]
Is anything further needed on this issue, or should we close it?

**Comment by josephdecock** [comment]
Closing, but feel free to reopen if necessary.
